db/migrate/20181013195028_add_confidential_to_applications.rb

```diff
@@ -0,0 +1,13 @@
+# frozen_string_literal: true
+
+class AddConfidentialToApplications < ActiveRecord::Migration
+  def change
+    add_column(
```

Comment:
can you use the style of the other migration files here too? (one line, no brackets)